### PR TITLE
PLF-6640 : LiquibaseDataInitializer starts too late.

### PR DESCRIPTION
**Pull request by aboughzela**
The service LiquibaseDataInitializer should be start and initialise data (Processing changelogs) at RootContainer Level.

### CODE REVIEW

File: commons-component-common/src/main/resources/conf/configuration.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+               xsi:schemaLocation="http://www.exoplatform.org/xml/ns/kernel_1_2.xsd http://www.exoplatform.org/xml/ns/kernel_1_2.xsd"
+               xmlns="http://www.exoplatform.org/xml/ns/kernel_1_2.xsd">
+   <component>
```
Comment (thomasdelhomenie) [inline commons-component-common/src/main/resources/conf/configuration.xml]: Can you add a comment here to explain why we need to declare this service in RootContainer as well as in PortalContainer please ?

File: commons-component-common/src/main/resources/conf/portal/configuration.xml [hunk 2]
```diff
@@ -55,7 +55,7 @@
     <type>org.exoplatform.commons.embedder.YoutubeEmbedder</type>
     <init-params>
       <value-param>
-        <name>(?:https?:\/\/)?(?:www\.)?(?:youtu\.be\/|youtube\.com\/(?:embed\/|v\/|watch\?v=|watch\?.+&amp;v=))((\w|-){11})(?:\S+)?$</name>
+        <name>(?:https?:\/\/)?(?:www\.)?(?:youtu\.be\/|youtube\.com\/(?:embed\/|v\/|watch\?v=|watch\?.+&amp;v=))((\w|-){11})(?:\S+)?$</name>
```
Comment (thomasdelhomenie) [inline commons-component-common/src/main/resources/conf/portal/configuration.xml]: What is the change ?

**Comment by aboughzela** [comment]
Add a comment ( why we need to declare this service in RootContainer)
Remove not useful change

**Comment by thomasdelhomenie** [comment]
LGTM

**Pull request closed without merging**